src/mainboard/sunxi/nezha/main/src/feature/emulate_rdtime.rs

```diff
@@ -9,8 +9,24 @@ const DEBUG_THIS: bool = false;
 
 #[inline]
 pub fn emulate_rdtime(ctx: &mut SupervisorContext, ins: usize) -> bool {
-    //  c0002573     rdcycle a0
-    if ins & 0xFFFFF07F == 0xC0002073 {
+    if ins & 0xFF0000FF == 0xf10000f3 {
```

Comment:
this and the following junk resulted from experimenting a bit - I shall remove it and see that we do this in a clean way at some point; it's quite wrong here at best